src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,17 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import core.mate.academy.model.Machine;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Machine> {
+
+    private Bulldozer firstBulldozer = new Bulldozer("Road", "Orange", 12, false);
+    private Bulldozer secondBulldozer = new Bulldozer("Construct", "Red", 22, true);
+    private Bulldozer thirdBulldozer = new Bulldozer("Additional", "Yellow", 18, true);

```

Comment:
lets make this variables local

Reply:
That's OK remove **private**, then Maven tell me: " Variable 'firstBulldozer' must be private and have accessor methods. [VisibilityModifier]"